utils/questGen.py

```diff
@@ -77,3 +72,104 @@ def pokemonname(id):
     with open('pokemon.json') as f:
         mondata = json.load(f)
     return mondata[str(int(id))]["name"]
+
+def questtask(typeid, condition, target):
+    with open('utils/quest/pokemonTypes.json') as f:
+        pokemonTypes = json.load(f)
+    with open('utils/quest/items.json') as f:
+        items = json.load(f)
+    throwTypes = {"10":"Nice", "11":"Great", "12":"Excellent", "13":"Curveball"}
```

Comment:
For consistency of having all this stuff in JSON file, I would expect this to be in a JSON file. Just easier to update if Nia make a change on throws for example

Reply:
Yeah actually Your right. Dunno why i didnt. Maybe because it were just 4 :D